src/main/java/ch/njol/skript/effects/EffHealth.java

```diff
@@ -73,8 +73,10 @@ protected void execute(Event event) {
 		double amount = 0;
 		if (this.amount != null) {
 			Number amountPostCheck = this.amount.getSingle(event);
-			if (amountPostCheck == null)
+			if (amountPostCheck == null) {
+				error("The provided " + getSyntaxType() + " amount was not set.", this.amount.toString());
```

Comment:
idk i have mixed feelings on the null value errors
maybe just postpone them for proper suppression